lstore/table.py

```diff
@@ -401,5 +399,12 @@ def write_record(self, record: Record, rid: int) -> bool:
         return True
 
 
-    def read_record(self) -> bool:
-        pass
+    def read_record(self, key):
+        record = Record(key, 0, "00000", [1, 2, 3, 4, 5, 6])
+        return record
+
+    def record_does_exist(self, key) -> bool:
+        return True
```

Comment:
This would just search the column for the key and if it doesn't find it return false? Would you want it to return the RID is it does exist though or is True enough for yyour use case?

Reply:
Yes, we would want the rid if true and False if record does not exist.